v2/pink-sb/src/lib/sidebar/Base.svelte

```diff
@@ -0,0 +1,193 @@
+<script lang="ts">
+    import { Badge, Icon } from '$lib/index.js';
+    import { IconChevronLeft, IconChevronRight, IconMinus } from '@appwrite.io/pink-icons-svelte';
+    import type { $$Props } from '$lib/sidebar/index.js';
+
+    export let state: $$Props['state'] = 'open';
+    export let resizable: $$Props['resizable'] = true;
+</script>
+
+{#if resizable}
+    <button
+        class="collapse"
+        class:icons={state === 'icons'}
+        on:click={() => {
+            state = state === 'icons' ? 'open' : 'icons';
+        }}
+    >
+        <div class="lines-container">
+            <div class="icon icon-idle minus-icon"><Icon icon={IconMinus} /></div>
+            <div class="icon icon-hover">
+                <Icon icon={state === 'icons' ? IconChevronRight : IconChevronLeft} />
+            </div>
+        </div>
+        <div class="badge">
+            <Badge
+                size="xs"
+                variant="primary"
+                content={state === 'icons' ? 'Expand' : 'Collapse'}
+            />
+        </div>
+    </button>
+{/if}
```

Comment:
I'm still seeing the scroll bars 
![image](https://github.com/user-attachments/assets/63035fa0-6d9d-4148-bce0-f6defe9ec29a)